includes/admin/class-wcmp-export.php

```diff
@@ -1143,14 +1143,40 @@ public function getShipmentData(array $ids, WC_Order $order): array
     }
 
     /**
+     * @param $weight
+     *
+     * @return float
+     */
+    public function getItemWeight_gram($weight): float
+    {
+        $weight_unit = get_option("woocommerce_weight_unit");
+        switch ($weight_unit) {
+            case "kg":
+                $grams = $weight * 1000;
+                break;
+            case "lbs":
+                $grams = $weight / 0.45359237;
+                break;
+            case "oz":
+                $grams = $weight / 0.0283495231;
+                break;
+            default:
+                $grams = $weight;
```

Comment:
Ik zou dit aan het begin van de functie zetten en die hele `default` weghalen, maar dat is gewoon een voorkeur dingetje. Ik vind het zelf leesbaarder om meteen de default value te zien, dan hoef je niet de hele switch/case door te lezen om die waarde te vinden.